lambdas/indexer/app.py

```diff
@@ -72,41 +66,16 @@ def version():
 @app.route('/health', methods=['GET'], cors=True)
 @app.route('/health/{keys}', methods=['GET'], cors=True)
 def health(keys: Optional[str] = None):
-    if keys == 'basic':
-        return {
-            'up': True,
-        }
-    elif keys == 'cached':
-        storage_service = StorageService()
-        health_object = json.loads(storage_service.get('health/indexer'))
-        max_age = 2 * 60
-        if time.time() - health_object['time'] > max_age:
-            return Response(status_code=500,
-                            body=json.dumps({'Message': 'Cached health object is stale'}))
-        else:
-            body = health_object['health']
-    else:
-        health = Health('indexer')
-        if keys is None:
-            if app.current_request.context['path'].endswith('/'):
-                keys = ()
-        else:
-            keys = keys.split(',')
-        try:
-            body = health.as_json(keys)
-        except RequirementError:
-            raise BadRequestError('Invalid health keys')
-    return Response(body=json.dumps(body),
-                    status_code=200 if body['up'] else 503)
+    controller = HealthController(lambda_name='indexer',
+                                  keys=keys,
+                                  request_path=(app.current_request.context['path']))
```

Comment:
```suggestion
                                  request_path=app.current_request.context['path'])
```

Reply:
Does your PyCharm not warn you about this? Mine usually does.

Reply:
No warning.

Reply:
Did you see https://humancellatlas.slack.com/archives/CGDQ4DNJY/p1570658294009600?